web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -166,8 +172,8 @@
 
           var unWatchFn = $rootScope.$watch(function() {
             return map.getTargetElement() && Math.min(
-              map.getTargetElement().offsetWidth,
-              map.getTargetElement().offsetHeight
+                map.getTargetElement().offsetWidth,
```

Comment:
Try not to change spaces if the line is not modified

Reply:
This was a fix made by closures' `fixjsstyle` a tool all we should run before commiting any changes.